src/Pmi/Controller/EvaluationController.php

```diff
@@ -143,6 +143,7 @@ public function evaluationAction($participantId, $evalId, Application $app, Requ
             $evaluationService->loadFromArray($evaluation);
             $evaluation['canCancel'] = $evaluationService->canCancel();
             $evaluation['canRestore'] = $evaluationService->canRestore();
+            $evaluation['reasonDisplayText'] = !empty($evaluation['eh_reason']) ? $evaluationService->getReasonDisplayText() : null;
```

Comment:
Instead of checking for an empty reason here, can you move that check into the `getReasonDisplayText` method?